src/sparseml/modifiers/quantization/gptq/base.py

```diff
@@ -53,6 +56,19 @@ class GPTQModifier(Modifier):
         in the recipe
     :param dampening_frac: Amount of dampening to apply to H, as a fraction of the
         diagonal norm
+    :param config_groups: [Used, if a quantization modifier is not specified],
+        dictionary specifying quantization schemes to apply to target
+        modules. Modules not matching a scheme target will NOT be quantized.
+    :param ignore: [Used, if a quantization modifier is not specified]
+        optional list of module class names or submodule names to not
+        quantize even if they match a target in config_groups. Defaults to empty list.
+    :param disable_quantization_observer_epoch: [Used, if a quantization modifier is
+        not specified] Epoch to disable updates to the module
+        quantization observers. At this point, quantized weights and zero points will
+        not be updated. Leave None to not disable observers during QAT. Default is None
+    :param num_calibration_steps: [Used, if a quantization modifier is not specified]
+        Number of steps to run post training calibration for.
+        When None, the entire calibration_dataloader is used
```

Comment:
This gets used regardless of if we specify a quant modifier since gptq also has a calibration step